src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/catchparametername/Example2.java

```diff
@@ -8,19 +8,25 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.naming.catchparametername;
+
+
+import java.io.IOException;
+
 // xdoc section -- start
-public class MyTest {
+public class Example2 {
   public void myTest() {
     try {
+      throw new IOException();
       // ...
     } catch (ArithmeticException ex) { // OK
       // ...
     } catch (ArrayIndexOutOfBoundsException ex2) { // OK
       // ...
     } catch (IOException thirdException) { // OK
       // ...
-    } catch (Exception FourthException) { // violation, the initial letter
-                                          // should be lowercase
+    } catch (Exception FourthException) {
```

Comment:
Related to the above, make sure that this example is in both, I do not see an upper CamelCase in example #1

Reply:
yes. fixed and pushed